File: TEST/COMPARE/9x9multiply_format/no_go_file/main.py
import time

def test():
    for i in range(1,10):
        for j in range(1,i+1):
            print(f"{j}x{i}={j*i}  ",end="")
        print()

N=3

if __name__ == "__main__":
    # test()
    # timeit.repeat(lambda : test(),repeat=3)
    start = time.time()
    for i in range(N):
        test()
    end = time.time()
    print(f"耗时是:{(end-start)/N}")

File: TEST/COMPARE/9x9multiply_format/no_go_file/main.jl
using BenchmarkTools

@time function test()
    for i = 1:9
        for j = 1:i
            print(i,"x",j,"=",i*j,"\t")
        end
        println()
    end
end

for i = 1:3
    test()
    # println(i)  
end